### PR TITLE
Fix pronoun ambiguity loopholes

### PR DESCRIPTION
**Pull request by liam-middlebrook**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [x] Non-semantic Change: Spelling, grammar, or formatting changes.

### CODE REVIEW

**Comment by mbillow** [comment]
I would consider this a non-semantic change. The rules have always applied to all genders.

**Comment by mbillow** [comment]
Also, we should make sure these are the only two instances of this, I feel like there are more.

**Comment by liam-middlebrook** [comment]
I searched for ' his' case insensitive and didn't see any other occurrences.

**Comment by mbillow** [comment]
There is also at least one instance of "he".

**Comment by liam-middlebrook** [comment]
That should be fixed now.

File: bylaws.tex [hunk 1]
```diff
@@ -253,9 +253,9 @@
 The Membership Evaluation is responsible for determining those individuals who will have the option to continue Active Membership in the following year.
 Semi-Annual Evaluations Meetings are open only to all current Active Members.
 All Executive Board members are expected to attend the Semi-Annual Evaluations Meetings to assist in the evaluations of House members.
-\\* \\*
-At the beginning of any Evaluation Process listed herein, the Evaluations Director must read the sections of the Computer Science House Constitution and By-Laws used during the respective Evaluation Process.
-It is incumbent upon each House member to provide the Evaluation Director with whatever information he or she feels is necessary to ensure an accurate evaluation.
+\v* \\*
+Anning of any Evaluation Process listed herein, the Evaluations Director must read the sections of the Computer Science House Constitution and By-Laws used during the respective Evaluation Process.
```
Comment (ajgajg1134) [inline bylaws.tex]: Is this change intentional?
